### PR TITLE
Add dockerfile to build geff image to run in lambda docker

### PR DESCRIPTION
**Pull request by sfc-gh-pkommini**
docker file to build geff as a docker image:

_**Testing**_:
abuse IP DB scan works:
image

ecr list_repos works:
image

With storage pipeline works:
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM public.ecr.aws/lambda/python:3.8
```
Comment (sfc-gh-afedorov) [inline Dockerfile:1]: would 3.10 be ok?

**Review by sfc-gh-afedorov — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (sfc-gh-pkommini) [reply Dockerfile:1]: Yeah I don't see any issues. Will test and push the change.

Comment (sfc-gh-pkommini) [reply Dockerfile:1]: Latest seems to be python3.9: https://docs.aws.amazon.com/lambda/latest/dg/python-image.html

Comment (sfc-gh-pkommini) [reply Dockerfile:1]: And it works just fine.

Comment (sfc-gh-afedorov) [reply Dockerfile:1]: Got it -- is there no public spec w.r.t. bases and do they all have to run using public.ecr.aws? py310 was released about a month ago, so it seems they have not yet gotten to releasing the lambda base for it.

**Pull request merged**